cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -217,15 +217,15 @@ public Future<Void> rollingRestart(Function<Pod, RestartReasons> podNeedsRestart
         this.podNeedsRestart = podNeedsRestart;
         Promise<Void> result = Promise.promise();
         singleExecutor.submit(() -> {
+            LOGGER.infoCr(reconciliation, "Verifying cluster pods are up-to-date.");
```

Comment:
Should this really be an INFO message? Wouldn't DEBUG be enough? My guess is that it will be printed everytime.

Reply:
It will indeed be printed everytime. My reasoning is that this message is also printed everytime in Info

```
INFO  AbstractOperator:239 - Reconciliation #49(timer) Kafka(my-kafka-project/my-cluster): Kafka my-cluster will be checked for creation or modification
```

So on Info, it's confusing without this initial message for the case when e.g. an exception happen when trying to roll broker-0 . It becomes like this

```
INFO  AbstractOperator:239 - Reconciliation #49(timer) Kafka(my-kafka-project/my-cluster): Kafka my-cluster will be checked for creation or modification
INFO  KafkaRoller:600 - Reconciliation #49(timer) Kafka(my-kafka-project/my-cluster): Will temporarily skip verifying pod my-cluster-0 is up-to-date due to : Exception ..... , retrying after at least 500ms.
INFO  KafkaRoller:600 - Reconciliation #49(timer) Kafka(my-kafka-project/my-cluster): Dynamic update of pod {my-cluster-kafka-0} was successful.
INFO  KafkaRoller:600 - Reconciliation #49(timer) Kafka(my-kafka-project/my-cluster): Dynamic update of pod {my-cluster-kafka-2} was successful.
INFO  KafkaRoller:600 - Reconciliation #49(timer) Kafka(my-kafka-project/my-cluster): Dynamic update of pod {my-cluster-kafka-1} was successful.
```

So adding clarifies that there is a rolling attempt going on.

Reply:
I'm not sure why would it be confusing. Nobody reported it before as confusing. But in any case, I do not have a problem with it being printed when an update happens. But if it is printed every reconciliation, it will be completely unnecessary when no update is happening.

Reply:
> I'm not sure why would it be confusing. Nobody reported it before as confusing.

IMHO if a user sees a log message that some operation failed , it's generally more informative if there's a previous message that shows which operation was being attempted. Without this message, this will lead to the higher level  ` Kafka my-cluster will be checked for creation or modification` message. 

However, I would've also only conditionally  logged it in case there was an update. But that's known at that level since that decision is taken much lower in the code and in different places.

But maybe indeed the overhead of adding a new line with every timer doesn't outweigh the benefit explained above. 
I will change it to DEBUG.